tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -430,7 +443,50 @@ Open New Notebook
     Sleep    1
     Maybe Close Popup
 
+Clone Repo
+    [Documentation]    It is a private keyword used by other keyword to clone the git repo
+    [Tags]    Private Keyword
+    [Arguments]    ${repo_url}
+    Navigate Home (Root folder) In JupyterLab Sidebar File Browser
+    Open With JupyterLab Menu    Git    Clone a Repository
+    Input Text    //div[.="Clone a repo"]/../div[contains(@class, "jp-Dialog-body")]//input    ${repo_url}
+    Click Element    xpath://div[.="CLONE"]
+
+
+Clone Repo and Return Error Message
+    [Documentation]    Clones the github repository and returns the error
+    [Tags]    Private Keyword
```

Comment:
It should be just `Private`.  

Additionally, you could add in the docs something like this:
[Documentation]    (Private Keyword) Clones the github repository and returns the error

Reply:
Thinking again, as the code has been tested and it has 2 approvals this can be changed in another PR